src/seeds.proposals.cpp

```diff
@@ -38,34 +38,61 @@ void proposals::reset() {
   }
 }
 
-bool proposals::is_enough_stake(asset staked, asset quantity) {
-  uint64_t min = min_stake(quantity);
+bool proposals::is_enough_stake(asset staked, asset quantity, name fund) {
+  uint64_t min = min_stake(quantity, fund);
   return staked.amount >= min;
 }
 
-uint64_t proposals::min_stake(asset quantity) {
-  auto prop_percentage = config.get(name("propstakeper").value, "The propstakeper parameter has not been initialized yet.");
-  auto prop_min = config.get(name("propminstake").value, "The propminstake has not been initialized yet.");
-  auto prop_max = config.get(name("propmaxstake").value, "The propmaxstake has not been initialized yet.");
-  asset quantity_prop_percentage = prop_percentage.value * quantity / 100;
+uint64_t proposals::cap_stake(name fund) {
+  uint64_t prop_max;
+  if (fund == bankaccts::campaigns) {
+    prop_max = config.get(name("prop.cmp.cap").value, "The prop.cmp.cap has not been initialized yet.").value;
+  } else if (fund == bankaccts::alliances) {
+    prop_max = config.get(name("prop.al.cap").value, "The prop.al.cap has not been initialized yet.").value;
+  } else {
+    prop_max = config.get(name("propmaxstake").value, "The propmaxstake has not been initialized yet.").value;
+  }
+  return prop_max;
+}
+
+uint64_t proposals::min_stake(asset quantity, name fund) {
+
+  double prop_percentage;
+  uint64_t prop_min;
+  uint64_t prop_max;
+
+  prop_min = config.get(name("propminstake").value, "The propminstake has not been initialized yet.").value;
+  
+  if (fund == bankaccts::campaigns) {
+    prop_percentage = 2.5;
```

Comment:
yes the % need to be settings too

Reply:
We could make them x 10,000 as integers, so 4 digits of precision like an asset. 

So 2.5% would be 25000 => 25,000 / 10,000 = 2.5 %

Reply:
@IanMendozaJaimes

Reply:
used to be called 
  confwithdesc(name("propstakeper"), 5, "Proposal funding fee in % - 5%", high_impact);

just need another one of those...